config/app/twig_hooks/admin/credit_memos/show.yaml

```
@@ -0,0 +1,252 @@
+sylius_twig_hooks:
+    hooks:
+        'sylius_refund.admin.credit_memos.show':
```

```suggestion
        'sylius_refund.admin.credit_memo.show':
```
and the same in other hooks